CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- [...]
+
+# v0.20.2 (02/01/2018)
```

Comment:
Happy new year!

Reply:
Fixed, but I suggest we automatize that changelog haha